langchain4j-dashscope/src/main/resources/META-INF/services/java.nio.file.spi.FileTypeDetector

```diff
@@ -0,0 +1 @@
+dev.langchain4j.model.dashscope.ImageFileTypeDetector
```

Comment:
The dashscope sdk directly uses java.nio.file.Files.probeContentType() to obtain the mimeType of a file. It relies on SPI to inject a specific FileTypeDetector implementation, otherwise it defaults to using sun.nio.fs.DefaultFileTypeDetector, which performs poorly.

The purpose of this file is to inject my own implementation of FileTypeDetector through SPI. It will be called by java.nio.file.Files.probeContentType.

Reply:
I see, thank you for explanation